<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>949</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="pgmMap">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>811</width>
     <height>821</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="gridMap">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>811</width>
     <height>821</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="collisionMap">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>390</y>
     <width>811</width>
     <height>821</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="pointMap">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>811</width>
     <height>821</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dataOpen">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dataSave">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QComboBox" name="requestTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="tableNumber">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="fileIndex">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dataClear">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dataCreate">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
